blog/experimental-support-for-gophers-in-wasmCloud/index.mdx

```diff
@@ -0,0 +1,66 @@
+description: "wasmCloud now comes with experimental component support for Go"
+categories: ["Golang", "wasmcloud"]
+draft: false
+---
+
+import ReactPlayer from 'react-player'
+
+![from wasmcloud specific dependencies to zero dependencies](./blobby-versus-globby.jpg)
+
+wasmCloud now comes with experimental support for Go, via TinyGo compiler, taking full advantage of the WebAssembly component model. This is an exciting moment as, for the first time, Go developers can build cloud native Wasm applications with components and run them in wasmCloud in an open, dependency-free and standards-based way.
+
+<!--truncate-->
+
+:::info
+
+*[TinyGo](https://tinygo.org/)* is *"a Go compiler for small places"*. It is a language designed specifically to work on embedded systems and WebAssembly. For Gophers, TinyGo is commonly a simple change from running `go build` to running `tinygo build`, as TinyGo [supports many of the Go standard library packages](https://tinygo.org/docs/reference/lang-support/stdlib/).
+
+:::
+
+Some of the best-known organizations develop in Go; Google, Uber, Netflix, Twitch to name a few. There's a growing appetite for Wasm amongst Go developers in all industries and so, with Go support now in wasmCloud, they have a community-driven, open environment to get WebAssembly projects off the ground.
+
+### Rebuilding Actors as Components
+
+wasmCloud has supported Go for [some time](https://wasmcloud.com/blog/example_creating_webassembly_actor_in_go_with_tinygo). As many in the community know, there are several TinyGo example Wasm applications, including `echo actor`, `blobby actor` and `kv-counter actor`, which were written around WASI-Preview 1 specifications. These examples compile to wasm modules and use our stable ABI and, as such, they rely on a few wasmCloud-specific dependencies to successfully run there.
+
+As wasmCloud is engineered around WASI-Preview 2, we have rewritten these actors to use WASI contracts; completely removing any dependencies. Smithy is a thing of the past as we focus on [WIT](https://cosmonic.com/blog/engineering/wit-cheat-sheet) to bind together polyglot components. Take a look at our new WASIfied apps `[echo-http-server](https://github.com/wasmCloud/wasmCloud/tree/main/examples/golang/actors/http-echo-tinygo)` and `[globby-wasi](https://github.com/jordan-rash/globby-wasi)` (blobby reimagined in Go!).
+
+This means developers can port their Wasm applications to wasmCloud without any additional requirements. It also means, when componentized, applications are even tinier than before.
+
+Long-time maintainer of wasmCloud Jordan Rash, and our own Roman Volosatovs, are working hard to bring a host of capabilities to WASI. Right now, these work with Version 7 of the host. By the time Wasmtime 15 comes out we're hoping to have moved all our examples to WASI. Meantime, let's take a closer look at what this means when developing Wasm applications for wasmCloud.
```

Comment:
```suggestion
Long-time maintainer of wasmCloud Jordan Rash, and host maintainer Roman Volosatovs, are working hard to bring a host of capabilities to WASI. Right now, these work with Version 7 of the host. By the time Wasmtime 15 comes out we're hoping to have moved all our examples to WASI. Meantime, let's take a closer look at what this means when developing Wasm applications for wasmCloud.
```

Reply:
What's version 7?

Reply:
Also, Wasmtime 15 landed today so let's strike the sentence about hoping to move all our examples 😅

Reply:
@brooksmtownsend please double check against Connor's question around the host.

Reply:
> Also, Wasmtime 15 landed today so let's strike the sentence about hoping to move all our examples 😅

Adjusted the copy per your advice @connorsmith256